eclipse-scout-core/test/form/fields/htmlfield/HtmlFieldSpec.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2010-2023 BSI Business Systems Integration AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     BSI Business Systems Integration AG - initial API and implementation
+ */
+import {FormSpecHelper} from '../../../../src/testing/index';
+import {HtmlField} from '../../../../src/index';
+
+describe('HtmlField', () => {
+  let session: SandboxSession, helper: FormSpecHelper;
+
+  beforeEach(() => {
+    setFixtures(sandbox());
+    session = sandboxSession();
+    helper = new FormSpecHelper(session);
+    jasmine.Ajax.install();
+    jasmine.clock().install();
+  });
+
+  afterEach(() => {
+    jasmine.clock().uninstall();
+    jasmine.Ajax.uninstall();
+  });
+
+  describe('acceptInput', () => {
+    it('does not change field value and displayText', () => {
+      let field = new HtmlField();
+      let model = helper.createFieldModel(HtmlField, null, {
```

Comment:
You don't have to create the model using the helper. Using scout.create(HtmlField, {
  parent: session.desktop,
  scrollBarEnabled: true
}
should be sufficient.